charts/testkube-dashboard/values.yaml

```diff
@@ -151,6 +151,13 @@ oauth2:
     #- --show-debug-on-error=true
   ## Array with extra environment variables
   extraEnvVars: []
+  ## Array with extra sources for environment variables
+  extraEnvFrom: []
```

Comment:
Could you please duplicate this var the `teskube/values.yaml` file?

Reply:
Done.  Chart versions have not been changed as I'm not sure of your policy or other locations needing a change for that.

Reply:
Chart version will be automatically bumped on merge to main so no need to so smth extra here.